### DISCUSSION TITLE
extract window.__NUXT__ to <script>

### DISCUSSION THREAD

**Discussion by iiemos** [nuxt/nuxt#17021] [Questions] [answered]
hello！ I want to extract window.NUXT to <script> tag，It's friendly to SEO. today I find https://nuxtjs.org/ has been implemented this function. please update the lastest doc. show how to config, thanks!

**Comment by pi0** [comment, marked as answer]
Hi. We are working on a similar feature for Nuxt 3 you can track in https://github.com/nuxt/framework/pull/6455

**Reply by iiemos** [reply]
thanks
